tests/averaging/rotation/test_shonan.py

```diff
@@ -34,11 +37,17 @@ def __execute_test(self, i2Ri1_input: Dict[Tuple[int, int], Rot3], wRi_expected:
         """Helper function to run the averagaing and assert w/ expected.
 
         Args:
-            i2Ri1_input: relative rotations, which are input to the algorithm.
-            wRi_expected: expected global rotations.
+            i2Ri1_input: Relative rotations, which are input to the algorithm.
+            wRi_expected: Expected global rotations.
         """
         i1Ti2_priors: Dict[Tuple[int, int], PosePrior] = {}
-        wRi_computed = self.obj.run_rotation_averaging(len(wRi_expected), i2Ri1_input, i1Ti2_priors)
+        two_view_estimation_reports = {
```

Comment:
could you add another test, keeping this one unchanged, (for default behavior that does not need two view reports)?